### DISCUSSION TITLE
Personnaliser le message pour les mesures d'audience

### DISCUSSION THREAD

**Discussion by Kracter56** [AmauriC/tarteaucitron.js#683] [Q&A]
Bonjour,

Je suis en train de configurer tarteaucitron pour un site web. J'aimerai personnaliser le message d'information associée aux mesures d'audience. 

Savez-vous comment paramétrer cela svp ? 

Autre question : comment peut-on modifier la couleur du bandeau et la taille de la police svp ? 

Merci pour votre retour.

<img width="862" alt="Capture d’écran 2021-04-29 à 11 18 26" src="https://user-images.githubusercontent.com/39699876/116530573-7ab7bd00-a8de-11eb-8d83-a4e67ea9b46a.png">

**Comment by AmauriC** [comment]
Pour personnaliser le texte, il faut ajouter, avant le chargement de tarteaucitron:

```
<script>
var tarteaucitronCustomText = {
    "analytic": {
        "details": "VOTRE TEXTE"
    },
};
</script>
```

Pour les couleurs, il faut le faire avec des règles CSS ;)

**Reply by Kracter56** [reply]
Merci pour ce retour rapide :) 

Là, je vois _tarteaucitronCustomText_ est-ce une variable "lambda" que vous avez considéré ou est-ce vraiment cette variable là qu'il faut modifier ? Je ne sais pas si ma demande est assez claire, je souhaite modifier le texte qui correspond à la catégorie "Mesure d'audience" (et aussi "Réseaux sociaux"), encadré en rouge dans mon screenshot, comment les personnaliser svp ? 

Merci d'avance.

**Reply by AmauriC** [reply]
Il faut vraiment utiliser cette syntaxe:
```
<script>
var tarteaucitronCustomText = {
    "analytic": {
        "details": "VOTRE DESCRIPTION ANALYTIC"
    },
    "social": {
        "details": "VOTRE DESCRIPTION RESEAUX SOCIAUX"
    },
};
</script>
```
